src/Components/Sidebar/sidebar.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import "./sidebar.css";
+import { Link } from "react-router-dom";
+
+const Sidebar = () => {
+  return (
+    <nav
+      className="sidebar video-sidebar"
+      style={{
+        marginLeft: "-10rem",
+        width: "14rem",
+      }}
+    >
+      <div>
+        <div
+          className="list-topic side-tags"
+          style={{ marginTop: "1rem", fontSize: "1.35rem" }}
+        >
+          <Link to="/">
+            <i class="fas fa-home" style={{ marginRight: "0.75rem" }}></i>
+            <span>Home</span>
+          </Link>
+        </div>
+      </div>
+
+      <div>
+        <div className="list-topic side-tags" style={{ fontSize: "1.35rem" }}>
+          <Link to="/playlist">
+            <i class="fas fa-play" style={{ marginRight: "0.75rem" }}></i>
+            <span>Playlist</span>
+          </Link>
+        </div>
+      </div>
+
+      <div>
```

Comment:
Also, I think, here these `div`s are getting repeated for the links, so you can use `map` here.

You can do something like
```javascript
[{
linkTo: "/watchlater",
icon: "clock",
linkText: "Watch Later"
}, ...objects for other links].map(link => (

<div>
        <div className="list-topic side-tags" style={{ fontSize: "1.35rem" }}>
          <Link to={link.linkTo}>
            <i className={`fas fa-${icon}`} style={{ marginRight: "0.75rem" }}></i>
            <span>{link.linkText}</span>
          </Link>
        </div>
      </div>

))
```

In this way, the number of lines of code will reduce a lot in your file.

Reply:
why I am saying this? Because the styling for all the links is same.

Reply:
Thankyou for your suggestions. Will surely implement that.